igvm/commands.py

```diff
@@ -328,13 +328,14 @@ def vm_build(vm_hostname, run_puppet=True, debug_puppet=False, postboot=None,
                     '"{}" is still running.'.format(vm.fqdn)
                 )
 
-            if rebuild and vm.hypervisor.vm_defined(vm):
+            if vm.hypervisor.vm_defined(vm):
```

Comment:
This invalidates the idea of "rebuild" being a different operation than "build". If somebody has a stopped VM, it will be undefined without asking and then building will fail because it's disk is still present.

Reply:
That is reasonable. I'll change that back.